### ISSUE TITLE
[FEAT]: Request for brighter "active selection" field

### ISSUE THREAD

**Issue by Philofil92** [nova-video-player/aos-AVP#756]
### Description

Hello everyone and support, I would like to know if it is possible to make a very small improvement (I think it's very easy to do and shouldn't take a lot of time!!!) let me explain. .. From an Android TV, when consulting a film or series file, the background of the file is never the same color, but roughly corresponds to the original color of the cover of the movie...Then from this file, you can select various options, reading, description, delete description...etc...etc... The problem is that there are not enough differences of color intensity between the choice of the selection field and the field selected "Active" and it happens too often that I select the wrong field!
I hope I was clear enough, if not, I want to deepen my question!
Long live Nova!!!

### Additional information

Nova Vidéo Player TV Android 11

**Comment by mitch133** [comment]
I completely agree, happens to me as well, particularly with red background and buttons.
If possible, could be a nice improvement indeed (if not at the API level, because I can observe same things on other apps)

**Comment by courville** [comment]
Changing DetailsActionButtonStyle focus saturation value seems easier than it is... Will try to find a way.

**Comment by Philofil92** [comment]
Hello, Yes, as they say, sometimes it's easier said than done.
See what you can do afterwards, it doesn't prevent Nova from working properly, 
it's just a request for a small improvement!!!
